test/suites/test_schema.py

```diff
@@ -336,14 +368,163 @@ def test_06_index_cached(self):
         # Verify that no schema fetches occurs.
         self.assertEqual(self.fetch_count, 0)
 
-    def test_07_schema_version_update(self):
+    def test_06_schema_version_update(self):
         _space_len = len(self.con.select('_space'))
         self.srv.admin("box.schema.create_space('ttt22')")
         self.assertEqual(len(self.con.select('_space')), _space_len + 1)
 
+    # For schema fetch disable testing purposes.
+    testing_methods = {
+        'unavailable': {
+            'replace': {
+                'input': ['tester', (1, None)],
+                'output': [[1, None]],
+            },
+             'delete': {
+                'input': ['tester', 1],
+                'output': [[1, None]],
+            },
+            'insert': {
+                'input': ['tester', (1, None)],
+                'output': [[1, None]],
+            }, 
+            'upsert': {
+                'input': ['tester', (1, None), []],
+                'output': [],
+            },
+            'update': {
+                'input': ['tester', 1, []],
+                'output': [[1, None]],
+            }, 
+            'select': {
+                'input': ['tester', 1],
+                'output': [[1, None]],
+            },
+            'space': {
+                'input': ['tester'],
+            },
+        },
+        'available': {
```

Comment:
Leave a comment here referring to `test_crud.py` regarding crud methods.

Reply:
Done